src/main/java/org/ow2/proactive/procci/request/CloudAutomationVariablesClient.java

```diff
@@ -14,23 +14,24 @@
 import org.springframework.stereotype.Service;
 
 /**
- * Created by mael on 06/10/16.
+ * Created by the Activeeon Team on 06/10/16.
  */
 
 /**
  * Send crud request on cloud-automation-service/variables
  */
 @Service
-public class DataServices {
+public class CloudAutomationVariablesClient {
 
-    private static final Logger logger = LogManager.getLogger(DataServices.class);
+    private static final Logger logger = LogManager.getLogger(CloudAutomationVariablesClient.class);
```

Comment:
Please Switch to SLF4J logging API and use parameterized messages:

http://slf4j.org/faq.html#paramException